library/sanitize.inc.php

```diff
@@ -77,3 +77,17 @@ function isWhiteFile($file)
 
     return false;
 }
+
+// check if a the first parameter contains a non-numeric value
+function sanitizeNumber($number,$name='')
+{
+    $clean_number = $number +0 ;
+
+    if ($clean_number==$number){
+        return $clean_number;
+    }
+    else {
+        error_log('Custom validation error: Parameter '.$name.' contains non-numeric value (A numeric value expected)');
```

Comment:
Lets make this really jump in the error_log and provide debugging info
``````
error_log('sanitizeNumber() OpenEMR ERROR: Value contains non-numeric value. Value: ' . $number . ' and was processed to: ' .  $clean_number);